src/main/java/com/minecolonies/coremod/inventory/api/CombinedItemHandler.java

```diff
@@ -38,27 +38,27 @@
     private       String                   customName;
 
     /**
-     * Method to create a new {@link AbstractCombinedItemHandler}.
+     * Method to create a new {@link CombinedItemHandler}.
      *
-     * @param defaultName The default name of this {@link AbstractCombinedItemHandler}.
+     * @param defaultName The default name of this {@link CombinedItemHandler}.
      * @param handlers    The combining {@link IItemHandlerModifiable}.
      */
-    protected AbstractCombinedItemHandler(@NotNull final String defaultName, @NotNull final IItemHandlerModifiable... handlers)
+    public CombinedItemHandler(@NotNull final String defaultName, @NotNull final IItemHandlerModifiable... handlers)
     {
         this.handlers = handlers;
         this.defaultName = defaultName;
     }
 
     /**
-     * Method to create a new combined {@link AbstractCombinedItemHandler} with
+     * Method to create a new combined {@link CombinedItemHandler} with
      * a given custom name.
      *
-     * @param defaultName The name of this {@link AbstractCombinedItemHandler}.
+     * @param defaultName The name of this {@link CombinedItemHandler}.
      * @param customName  The preset custom name of this {@link
-     *                    AbstractCombinedItemHandler}.
+     *                    CombinedItemHandler}.
      * @param handlers    The combinging {@link IItemHandlerModifiable}.
      */
-    protected AbstractCombinedItemHandler(@NotNull final String defaultName, @NotNull final String customName, @NotNull final IItemHandlerModifiable... handlers)
+    public CombinedItemHandler(@NotNull final String defaultName, @NotNull final String customName, @NotNull final IItemHandlerModifiable... handlers)
     {
         this.handlers = handlers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "handlers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)